nixos/modules/services/networking/supplicant.nix

```diff
@@ -1,42 +1,40 @@
 { config, lib, utils, pkgs, ... }:
-
-with lib;
-
 let
 
   cfg = config.networking.supplicant;
 
-  # We must escape interfaces due to the systemd interpretation
+  # We must lib.escape interfaces due to the systemd interpretation
```

Comment:
I don't think you meant to put `lib` here

Reply:
resolved